src/dataviews/dataview-model-base.js

```diff
@@ -100,8 +106,8 @@ module.exports = Model.extend({
 
   _initBinds: function () {
     this.listenTo(this.layer, 'change:visible', this._onLayerVisibilityChanged);
-    this.listenTo(this.layer, 'change:source', this._setupAnalysisStatusEvents);
-    this.on('change:source', this._setupAnalysisStatusEvents, this);
+    this.listenTo(this.layer, 'change:source', this._onLayerSourceChanged);
+    this.on('change:source', this._onSourceChanged, this);
```

Comment:
💡  We could also leave existing bindings as they were and add another one with a comment explaining why we're adding this:
```
// Temporary code to log changes to dataview's sources.
// TODO: To be removed after xxx is merged.
this.on('change:source', this._logSourceChange, this);
```

Reply:
:+1: